src/tool/cppwinrt/test_component_fast/test_component_fast.idl

```diff
@@ -0,0 +1,157 @@
+import "Windows.Foundation.idl";
+
+namespace Windows.Foundation.Metadata
+{
+    [attributeusage(target_method, target_property)]
+    [attributename("noexcept")]
+    attribute NoExceptionAttribute
+    {
+    }
```

Comment:
this isn't in midlrt yet?

Reply:
Only in APT7's version of MIDLRT - the build server won't have it for a while.